package.json

```diff
@@ -22,9 +22,11 @@
     "@types/chart.js": "^2.9.41",
     "astro": "^4.2.4",
     "chart.js": "^4.4.1",
+    "chartjs-adapter-luxon": "^1.3.1",
```

Comment:
related to robert comment, luxon is really huge dependency, alternative is https://github.com/chartjs/chartjs-adapter-date-fns which is tiny which I think is acceptable

@robertvanhoesel WDYT?

Reply:
TBH I really think preparing data for rendering in a chart is best done outside of the Chart component. This way we can keep the reusability of the component higher (i.e. could also display non-time series). 

`Intl.DateTimeFormat` is good enough, and especially with server prepared and rendered info a good way to go. 

If _really_ there's no way around it we can have a dependency on formatting outside of the chart rendering, this way server side we can do the formatting without the need of bundling and shipping the library to the client.